Composer/packages/server/src/controllers/formDialog.ts

```diff
@@ -1,12 +1,36 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import * as path from 'path';
+
```

Comment:
Nit: Extra space

Reply:
It's prettier + eslint separating thise imports